pyproject.toml

```diff
@@ -39,14 +40,16 @@ select = [
     "PYI032", # use `object`, not `Any`, as the second parameter to `__eq__`
     "PYI055", # multiple `type[T]` usages in a union
     "PYI058", # use `Iterator` as the return type for `__iter__` methods
-    "UP004", # Remove explicit `object` inheritance
-    "UP006", # PEP-585 autofixes
-    "UP007", # PEP-604 autofixes
-    "UP013", # Class-based syntax for TypedDicts
-    "UP014", # Class-based syntax for NamedTuples
-    "UP019", # Use str over typing.Text
-    "UP035", # import from typing, not typing_extensions, wherever possible
-    "UP039", # don't use parens after a class definition with no bases
+]
+ignore = [
+    # We like explicit open modes
```

Comment:
Is that even true? I know I do, because of the different open modes in different languages, one's expectation could be different.
The only place this is flagged in typeshed's codebase is in a stub test
https://github.com/python/typeshed/blob/5e9589dd75f8a5f39a38a2ed45f098ff62babc7f/stubs/requests/%40tests/test_cases/check_post.py#L46

Reply:
I think it might be a good idea to exclude all test cases from these rules. Test cases often deliberately contain code that might not be considered idiomatic or modern; I think it would be confusing (especially to new contributors) to have that incorrectly autofixed